webapp/components/Comment/Comment.spec.js

```diff
@@ -32,6 +32,9 @@ describe('Comment.vue', () => {
         truncate: a => a,
         removeHtml: a => a,
       },
+      $route: {
+        hash: '',
```

Comment:
Would be obsolete, if `Comment.vue` wouldn't access `this.$route`.

Reply:
true – thanks for pushing up your solution, I like it! :blush: